config/crd/common/bases/services.k8s.aws_fieldexports.yaml

```diff
@@ -3,7 +3,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
You need to install controller-gen v0.14.0

Reply:
Done